azurerm/internal/services/cognitive/resource_arm_cognitive_account.go

```diff
@@ -98,6 +98,22 @@ func resourceArmCognitiveAccount() *schema.Resource {
 				}, false),
 			},
 
+			"api_properties": {
```

Comment:
Just a random thought, but what do you think about removing this block and just having the sub properties at the top level? it would simplify the code, remove the expand/flatten functions, and provider a cleaner UX. I don't know if grouping them under `api_properties` really does much?

WDYT?

Reply:
yeah very good point, I left it in the api_properties block to keep parity with the SDK however I agree it will be cleaner at the top level. I will update the code

Reply:
done